src/components/Common/Fields/CheckboxField/CheckboxField.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import type { Story } from '@ladle/react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { CheckboxField } from './CheckboxField'
+
+interface FormWrapperProps {
+  children: React.ReactNode
+  defaultValues?: {
+    acceptTerms?: boolean
+    subscribeNewsletter?: boolean
+    enableNotifications?: boolean
+  }
+}
```

Comment:
Can we get some actions in here so we can validate the field is working from within ladle? Take a look how I did SelectField stories. This was a callout from @scally and I think it makes a big difference for usability.